server/directory.go

```diff
@@ -0,0 +1,20 @@
+package server
+
+import "github.com/coniks-sys/coniks-go/merkletree"
+
```

Comment:
I answered in https://github.com/coniks-sys/coniks-go/pull/29#discussion_r71993140